app/code/GumennoyEL/FirstModule/Controller/ShowPerson/Index.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace GumennoyEL\FirstModule\Controller\ShowPerson;
+
+use Magento\Framework\App\Action\Context;
+use Magento\Framework\App\ResponseInterface;
+use Magento\Framework\Controller\ResultFactory;
+
+class Index extends \Magento\Framework\App\Action\Action {
+
+	/**
+	 * @return \Magento\Framework\App\ResponseInterface|\Magento\Framework\Controller\ResultInterface
+	 */
+
+	public function execute() {
```

Comment:
Move open curly brace to the new line. Methods functions should not have open braces on the same line
```
public function execute() 
{
...
}
```